README.md

```diff
@@ -36,16 +36,13 @@ These tables are designed to replicate common GA4 reports. The following provide
 ### Step 1: Prerequisites
 To use this dbt package, you must have the following:
 
-- At least one Fivetran GA4 Export connector syncing data into your destination.
+- At least one [Fivetran GA4 Export](https://fivetran.com/docs/connectors/applications/google-analytics-4-export#googleanalytics4export) connector syncing data into your destination.
 - A **BigQuery**, **Snowflake**, **Redshift**, **PostgreSQL**, or **Databricks** destination.
 
-#### Databricks dispatch configuration
-If you are using a Databricks destination with this package, you must add the following (or a variation of the following) dispatch configuration within your `dbt_project.yml`. This is required in order for the package to accurately search for macros within the `dbt-labs/spark_utils` then the `dbt-labs/dbt_utils` packages respectively.
-```yml
-dispatch:
-  - macro_namespace: dbt_utils
-    search_order: ['spark_utils', 'dbt_utils']
-```
```

Comment:
Similar note that this is not necessary since we don't install the spark utils package.